import-automation/progress-dashboard-rest/app/utils.py

```diff
@@ -55,6 +60,33 @@ def setup_logging():
 
     Only logs at INFO level or higher will be captured.
     """
-    client = google.cloud.logging.Client()
+    client = logging.Client()
     client.get_default_handler()
     client.setup_logging()
+
+
+def create_storage_bucket(project=configs.PROJECT_ID,
```

Comment:
Does this always succeed?

Reply:
bucket() always succeeds. It does not make HTTP request according to https://googleapis.dev/python/storage/latest/client.html#google.cloud.storage.client.Client.bucket. Client() can fail if credentials are not found or the project ID is not valid